install.js

```diff
@@ -132,7 +132,9 @@ function findSuitableTempDirectory() {
 
   for (let i = 0; i < candidateTmpDirs.length; i++) {
     if (!candidateTmpDirs[i]) continue;
-    const candidatePath = path.join(candidateTmpDirs[i], 'chromedriver');
+    // Prevent collision with other versions in the dependency tree
+    const namespace = require('./package.json').version;
+    const candidatePath = path.join(candidateTmpDirs[i], namespace, 'chromedriver');
```

Comment:
Maybe not for this PR, but consider outsourcing `candidateTmpDirs` tempfile creation to a package like [tmp](https://www.npmjs.com/package/tmp). It's secure. It's multiprocessing safe. It's one less thing to maintain. 😄

Reply:
@john-kurkowski I'll check it out. Tks.